src/App.js

```diff
@@ -11,8 +11,15 @@ import NavBar from "./components/NavBar";
 import { UserProvider } from "./Auth";
 import RootContainer from "./components/RootContainer";
 import { SessionProvider } from "./Session";
+import ReactGA from 'react-ga';
 
 function App() {
+
+  useEffect(() => {
+    ReactGA.initialize('UA-165437579-1');
+    ReactGA.pageview(window.location.pathname + window.location.search);
+  });
```

Comment:
add empty dependency array for useEffect?

Reply:
done.